### PR TITLE
Update vda-subscription-activation.md

### PR DESCRIPTION
**Pull request by rybaker-microsoft**
First - eliminated QMTH references since that is no longer a valid program. All customers with E3 licenses are contractually allowed to run their WIndows Client VMs in public/shared clouds now. There is no longer a requirement that it happen on a "QMTH". 

Second, I found the scenarios confusing and I think there are only two: 1)  sign-in activation, and 2) inherited activation. So I updated the scenarios accordingly. Will you verify or have Michael Pashniak verify? Thanks!

## Why

- Closes #[Issue Number]

### CODE REVIEW

**Comment by frankroj** [comment]
Review against the following two PRs where recent changes were made to this same article in the same section:

https://github.com/MicrosoftDocs/windows-itpro-docs/pull/11289
https://github.com/MicrosoftDocs/windows-docs-pr/pull/8062

File: windows/deployment/vda-subscription-activation.md [hunk 1]
```diff
@@ -31,30 +31,24 @@ Deployment instructions are provided for the following scenarios:
...
 
-### Scenario 3
-
-- The hoster isn't an authorized QMTH partner.
-
-    In this scenario, the underlying Windows Pro license must be activated prior to using subscription activation Windows Enterprise. Activation is accomplished using a generic volume license key (GVLK) and a volume license KMS activation server provided by the hoster. Alternatively, a KMS activation server can be used. KMS activation is provided for Azure VMs. For more information, see [Troubleshoot Azure Windows virtual machine activation problems](/troubleshoot/azure/virtual-machines/troubleshoot-activation-problems).
-
-For examples of activation issues, see [Troubleshoot the user experience](./deploy-enterprise-licenses.md#troubleshoot-the-user-experience).
+For examples of activation issues, see [Troubleshoot the user experience](./deploy-enterprise-licenses.md#troubleshoot-the-user-experience) and see [Troubleshoot Azure Windows virtual machine activation problems](/troubleshoot/azure/virtual-machines/troubleshoot-activation-problems).
```
Comment (JohanFreelancer9) [inline windows/deployment/vda-subscription-activation.md:51]:
```suggestion
For examples of activation issues, see [Troubleshoot the user experience](./deploy-enterprise-licenses.md#troubleshoot-the-user-experience) and [Troubleshoot Azure Windows virtual machine activation problems](/troubleshoot/azure/virtual-machines/troubleshoot-activation-problems).

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @rybaker-microsoft, kindly commit the suggested minor change. Many thanks.

**Comment by frankroj** [comment]
Followed up with contributor since there were some outstanding items regarding the changes he made.

**Comment by frankroj** [comment]
I haven't heard back from contributor. Emailed him if its ok to close PR until we can figure out what the exact wording should be.

**Comment by frankroj** [comment]
Never heard back from submitter. Closing PR.

**Pull request closed without merging**